action.yaml

```diff
@@ -108,7 +108,7 @@ runs:
   using: 'composite'
   steps:
     - name: Install Trivy
-      uses: aquasecurity/setup-trivy@v0.1.0
+      uses: DmitriyLewen/setup-trivy@f4134be94c08be2fb699a60627177c300876eb3d
```

Comment:
```suggestion
      uses: aquasecurity/setup-trivy@v0.2.0
```

Reply:
updated in https://github.com/aquasecurity/trivy-action/pull/411/commits/cdae8f71fc2fc231480d1fa7dc64698c39f610f2